core/modules/qana/PostPlugin.cc

```diff
@@ -75,8 +75,14 @@ class PostPlugin : public QueryPlugin {
 
     int _limit;
     std::shared_ptr<query::OrderByClause> _orderBy;
+
+private:
+	static LOG_LOGGER logger;
 };
```

Comment:
I know you introduced this pattern in the past, but I just wonder... are we sure this code is thread safe? (does it need to be?)